README.md

```diff
@@ -25,16 +25,31 @@ The MyDRTV platform is designed to provide:
 ## Architectural Approach
 
 ### Architectural Styles Comparison
-We considered multiple architectural styles for the MyDRTV project. Below is a summary of the criteria used in our comparison:
+We considered multiple architectural styles for the MyDRTV project. Below is a summary of the criteria used in our comparison: 
 
 ## Technology Stack
 
-- **Frontend:** [Frontend technology, e.g., React, Angular]
-- **Backend:** [Backend technology, e.g., Node.js, Django]
-- **Database:** [Database, e.g., PostgreSQL, MongoDB]
-- **Search Engine:** [Search technology, e.g., Elasticsearch]
-- **Recommendation Engine:** [Algorithm or tech for recommendations]
-- **Infrastructure:** Cloud-hosted on [e.g., AWS, Azure] to ensure scalability and high availability.
+- **Frontend:** Typescript React. This choose is made over Javascript becuase we would like strongly typed language and gives better tooling then Javascript. Then React was choosen because it makes coding dynamic applications easier to write and more performant aswell as when we're already familiar with react beforehand.
+
+- **Backend:** We have choosen C# because its obejct-oriented and uses dotnet which gives access to Entity Framework (EF). EF is a object-relation mapper which makes it easier to make CRUD operations in a relational database. Other programming langauges also have object-relation mapper like JPA for Java but we like how easy and intuative EF is to use.
+
+- **Database:** We think PostgreSQL is a suitable choice because it is a relational database and we think i simple to use.
```

Comment:
```suggestion
- **Database:** We believe PostgreSQL is a suitable choice because it is a relational database and we find it simple to use, Postgres is also known for its scalability and ability to handle large-scale data. It also offers JSON support giving us the flexibility we want combined with a decent community support, making this a dependable choice for managing our data needs we'd say.
```